metadata/schema.go

```diff
@@ -52,12 +52,12 @@ func getSchema(field reflect.Type, components *ComponentMetadata, nested bool) (
 	return schema, nil
 }
 
-func buildArraySchema(array reflect.Value, components *ComponentMetadata) (*spec.Schema, error) {
+func buildArraySchema(array reflect.Value, components *ComponentMetadata, nested bool) (*spec.Schema, error) {
 	if array.Len() < 1 {
 		return nil, fmt.Errorf("Arrays must have length greater than 0")
 	}
 
-	lowerSchema, err := GetSchema(array.Index(0).Type(), components)
+	lowerSchema, err := getSchema(array.Index(0).Type(), components, nested)
```

Comment:
why does this change include changing the case on getSchema?

Reply:
It changes case as it now needs to pass whether it is nested or not the public GetSchema calls getSchema but with nested always false